doc/api/offsetbox_api.rst

```diff
@@ -2,6 +2,9 @@
 ``matplotlib.offsetbox``
 ************************
 
+.. inheritance-diagram:: matplotlib.offsetbox.OffsetBox matplotlib.offsetbox.PackerBase matplotlib.offsetbox.VPacker matplotlib.offsetbox.HPacker matplotlib.offsetbox.PaddedBox matplotlib.offsetbox.DrawingArea matplotlib.offsetbox.TextArea matplotlib.offsetbox.AuxTransformBox matplotlib.offsetbox.AnchoredOffsetbox matplotlib.offsetbox.AnchoredText matplotlib.offsetbox.OffsetImage matplotlib.offsetbox.AnnotationBbox matplotlib.offsetbox.DraggableBase matplotlib.offsetbox.DraggableOffsetBox matplotlib.offsetbox.DraggableAnnotation
```

Comment:
does it not work to just write `.. inheritance-diagram:: matplotlib.offsetbox`?  (as in dates_api.rst)

Reply:
Indeed. Changed.